backend/src/main/scala/bloop/util/Base64Uuid.scala

```diff
@@ -0,0 +1,27 @@
+package bloop.util
+import java.nio.ByteBuffer
+import java.util.{Base64, UUID}
+
+object Base64Uuid {
+
+  // Copied from https://stackoverflow.com/a/29836273
+  implicit class UuidWithBase64(val uuid: UUID) extends AnyVal{
```

Comment:
I propose to turn this into a method, there's no need for a value class

Reply:
Done